src/test/scala/com/mongodb/spark/AuthConnectionSpec.scala

```diff
@@ -1,50 +0,0 @@
-/*
```

Comment:
This file actually didn't test anything new, the whole test suite is run under auth.